### ISSUE TITLE
Support for windows

### ISSUE THREAD

**Issue by chennqqi** [reconquest/barely#2]
demo code run crash under windows-386...

**Comment by kovetskiy** [comment]
@chennqqi pull-requests are welcome

**Comment by seletskiy** [comment]
@chennqqi: I agree with @kovetskiy. I do not use windows by myself. So, if you have an opportunity to prepare an PR, I will gladly merge it.

**Comment by violarulan** [comment]
Well, you can run it in emulators(like [Cmder](http://cmder.net/))

![image](https://user-images.githubusercontent.com/10095657/28759226-2a950460-75cd-11e7-9e17-4179f7ac380f.png)
